lib/geoengineer/resources/aws/api_gateway/aws_api_gateway_resource.rb

```diff
@@ -14,9 +14,10 @@ class GeoEngineer::Resources::AwsApiGatewayResource < GeoEngineer::Resource
   after :initialize, -> { self.rest_api_id = _rest_api.to_ref }
   after :initialize, -> { _rest_api.api_resources[self.type][self.id] = self }
 
-  # Parent is a resource (the default being the root resource of the rest_api '/')
-  # At the moment we only support rest_api being the root resource
```

Comment:
@grahamjenson is there a good reason we were only supporting the root resource? From what I can tell with a quick look, we don't necessarily need to require it.